process/sig-lifecycle.md

```diff
@@ -1,13 +1,30 @@
 # Special Interest Group (SIG) Life Cycle
 
-Special Interest Groups are under the direct governance of a Working Group (WG) which can create and close them as necessary and appropriate. SIGs are expected to have a very specific goals and objectives at the time of their creation which may or may not define if they are terminable and is entirely dependent on the nature of the SIG.
+Special Interest Groups are under the direct governance of their reporting Working Group (WG) and are bound to achieving a very specific goal. These groups may be terminated upon completion of their designating tasking, continued for larger and ongoing efforts, or otherwise subject to the governance, structure, and termination policies of the WG they are under. The creation of a SIG must dictate the focus, intent, goals, and deliverables as appropriate.
 
-The lifecycle of SIGs is therefore minimal and as follows.
+It is expected that the primary output of a SIG is not software. If the primary output is software, the work should be organized as a [project](./project-lifecycle.md).
 
-## Active
+SIG process should be minimal with as much as possible delegated to the WG. However we do need minimal process to at least ensure we have an accurate list of all the active SIGs in the OpenSSF. This document uses "must" to describe what items are required and "should" to suggest items that should be strongly considered (but not required).
 
-* The SIG is actively pursuing its goal.
+## To become `Tentative`:
 
-## Inactive
+* This is the default state of a new SIG that is not yet active
 
-* The SIG is no longer active, either it has completed its mission or the mission was abandoned. At which state, the WG the SIG is under will determine a termination status and record it within the corresponding repository.
+## To become `Active`:
+
+* A SIG must have a repository (prefixed with `sig-`) that includes the current state of the SIG, lists the chairs of the SIG, and a statement of the focus, intent, goals, and/or deliverables of the group
+* The chairs of a WG must agree to govern the SIG
+  * The chairs of the WG should have its membership vote, with notice given at the meeting prior to the vote
+* The governing WG must list the active SIGs it governs, along with chairs and links to the repository of those SGs
+  * The governing WG should list previous SIGs that are now inactive
+* SIGs should have regular meetings
```

Comment:
Do we really want to require that?? This would mean a SIG can't just work asynchronously over slack and github for instance. I don't think that's right.
We do need to require some sustained activity so that we don't end up with zombie SIGs lying around but I don't think having meetings should be the right metric for this.

Reply:
The "should" means that this is a recommendation and not a requirement - maybe the "should" should be more clear? Otherwise I'm happy to strike this item altogether if we don't think it should be a recommendation.

Reply:
I don't think this is strictly necessary for a SIG to be successful.